scripts/generateAdBlockRustDataFiles.js

```diff
@@ -173,13 +175,17 @@ const generateTestDataFile4 =
 // 9852EFC4-99E4-4F2D-A915-9C3196C7A1DE/rs-9852EFC4-99E4-4F2D-A915-9C3196C7A1DE.dat
 const generateTestDataFile5 =
   generateDataFileFromLists.bind(null, [{ format: FilterFormat.STANDARD, data: 'ad_fr.png' }], 'rs-9852EFC4-99E4-4F2D-A915-9C3196C7A1DE.dat', 'test-data')
+// for iOS team - compile cosmetic filters only
+const generateTestDataFile6 =
+  generateDataFileFromLists.bind(null, [{rule_types: RuleTypes.COSMETIC_ONLY}], 'ios-cosmetic-filters.dat')
```

Comment:
You're missing the `data` property of the object in the array, which is supposed to specify rules to add. Although this is probably not the best place to add it, since the `generateTestDataFile*` functions have one or two filter rules each. To be honest I'm not even sure what they're really used for :sweat_smile:

I'd recommend adding to `generateDataFilesForList`, which already is handling the default lists (although it's named somewhat confusingly). If you add another `generateDataFileFromLists` call after adding `rule_types: RuleTypes.COSMETIC_ONLY` to everything in `listBuffers`, that should work pretty well.

Reply:
That worked perfectly! 😄